scripts/packaging/createupmpackages.ps1

```diff
@@ -159,11 +159,29 @@ foreach ($entry in $packages.GetEnumerator()) {
         $registryName = "the $OutputTarget registry"
     }
 
-    # The examples folder needs a custom setup step
-    if ($entry.Name -eq "examples") {
-        Write-Output "Preparing the examples folder"
+    $samplesFolder = "$packagePath\Samples~"
+     
+    if ($packageName -eq "examples") {
+        # The examples folder needs a custom setup step
         Start-Process -FilePath "$PSHOME\powershell.exe" -ArgumentList "$scriptPath\examplesfolderpreupm.ps1" -NoNewWindow -Wait
     }
+    else {
+        # Some other folders have localized examples that need to be prepared. Intentionally skip the foundation as those samples
+        # are pacakged in the examples package.
+        $exampleFolder = "$packagePath\Examples"
+        if (($PackageName -ne "foundation") -and (Test-Path -Path $exampleFolder)) {
+            if (Test-Path -Path $exampleFolder) {
+                # Ensure the required samples exists
+                if (-not (Test-Path -Path $samplesFolder)) {
+                    New-Item $samplesFolder -ItemType Directory | Out-Null

```

Comment:
I haven't seen the Out-Null thing that often before - what happens if it is removed?

Reply:
silences messages like

    Directory: C:\handPhysUpm\Assets\MRTK\Extensions\HandPhysicsService


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----        7/20/2020  11:00 AM                Samples~

Reply:
granted, no one is likely to watch this output. I do prefer a cleaner log